### ISSUE TITLE
POST to E1_HelloSequence triggers breakpoing in HttpStart

### ISSUE THREAD

**Issue by mmisztal1980** [Azure/azure-functions-durable-extension#7]
- I've set a breakpoint in the **HttpStart** Function
- I'm sending a `POST localhost:7071/orchestrators/E1_HelloSequence`
- The breakpoint in **HttpStart** has been hit before the execution reached **E1_HelloSequence**

(next I've seen the Webhooks not configured exception message)

Is this expected?

**Comment by cgillum** [comment]
That's correct. Hitting the breakpoint is expected. The orchestration gets started inside the HttpStart function [here](https://github.com/Azure/azure-functions-durable-extension/blob/master/samples/precompiled/HttpStart.cs#L23), which I'm guessing is just the line after your breakpoint.

See the [StartNewAsync](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationClient.html#Microsoft_Azure_WebJobs_DurableOrchestrationClient_StartNewAsync_System_String_System_Object_) documentation for more info.

**Comment by mmisztal1980** [comment]
Unfortunately `CreateCheckStatusResponse(req, instanceId)` throws the Webhooks not configured exception, which is a showstopper currently.

**Comment by cgillum** [comment]
Yep. I'm working on that now. I believe I've identified the issue (a missing app setting). I'll update that issue shortly.

**Comment by cgillum** [comment]
See the workaround for the `CreateCheckStatusResponse` issue here: https://github.com/Azure/azure-functions-durable-extension/issues/6
